### PR TITLE
Lua functions tests

### PR DESCRIPTION
**Pull request by CorentinLGS**
Added last test functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,21 +349,34 @@ callback let you add assertions and enrich the test.
     message is caracterized by its *api* and the message level *error*,
     *warning*, *notice*, *info* or *debug*.
 
-    The timeout argument is in seconds and could be omitted if you doesn't need
-    to wait to receive the event.
+* **_AFT.testEvtNotReceived(testName, eventName)**
```
Comment (claneys) [inline README.md]: Add a timeout event as last arguments even if not implemented right now, it shouldn't be long. So for one time we have the documentation more advanced than the code...

File: README.md [hunk 2]
```diff
@@ -311,21 +349,34 @@ callback let you add assertions and enrich the test.
...
+    Prior to be able to check that an event has been received, you have to
+    register the event with the test framework using **_AFT.addEventToMonitor**
+    function.
+
+    Check if an event has been correctly received. An event name use the
+    application framework naming scheme: **api/event_name**.
 
-> **NOTE**: about now this timeout arguments simply wait a number of seconds
-> instead of interrupt as soon as the event has been received.
+* **_AFT.testLogNotReceived(testName, logMsg)**
```
Comment (claneys) [inline README.md]: To be removed. Log are not managed anymore. :)

File: README.md [hunk 3]
```diff
@@ -295,13 +339,7 @@ callback let you add assertions and enrich the test.
...
     application framework naming scheme: **api/event_name**.
 
-    The timeout argument is in seconds and could be omitted if you doesn't need
-    to wait to receive the event.
-
-> **NOTE**: about now this timeout arguments simply wait a number of seconds
-> instead of interrupt as soon as the event has been received.
-
-* **_AFT.testLogReceived(testName, logMsg, timeout)**
+* **_AFT.testLogReceived(testName, logMsg)**
```
Comment (claneys) [inline README.md]: To be removed, log not managed anymore.

File: README.md [hunk 4]
```diff
@@ -272,21 +316,21 @@ callback let you add assertions and enrich the test.
...
 
-    If you doesn't need to test the response simply specify an empty LUA table.
+    If you don't need to test the response simply specify an empty LUA table.
 
 * **_AFT.testVerbCbError(testName, api, verb, args, expectedResponse, callback)**
 
     Should return success on failure.
 
-* **_AFT.testEvtReceived(testName, eventName, timeout)**
+* **_AFT.testEvtReceived(testName, eventName)**
```
Comment (claneys) [inline README.md]: Add a timeout event as last arguments even if not implemented right now, it shouldn't be long. So for one time we have the documentation more advanced than the code...

File: README.md [hunk 5]
```diff
@@ -372,12 +417,24 @@ callback let you add assertions and enrich the test.
     Check if a log message sent by the daemon has been correctly received. A log
     message is caracterized by its *api* and the message level *error*,
     *warning*, *notice*, *info* or *debug*.
+* **_AFT.assertEvtNotReceived(eventName)**
```
Comment (claneys) [inline README.md]: Add a timeout event as last arguments even if not implemented right now, it shouldn't be long. So for one time we have the documentation more advanced than the code...

File: README.md [hunk 6]
```diff
@@ -357,13 +408,7 @@ callback let you add assertions and enrich the test.
...
     application framework naming scheme: **api/event_name**.
 
-    The timeout argument is in seconds and could be omitted if you doesn't need
-    to wait to receive the event.
-
-> **NOTE**: about now this timeout arguments simply wait a number of seconds
-> instead of interrupt as soon as the event has been received.
-
-* **_AFT.assertLogReceived(testName, logMsg, timeout)**
+* **_AFT.assertLogReceived(testName, logMsg)**
```
Comment (claneys) [inline README.md]: To be removed. Log are not managed anymore. :)

File: README.md [hunk 7]
```diff
@@ -372,12 +417,24 @@ callback let you add assertions and enrich the test.
...
+    Prior to be able to check that an event has been received, you have to
+    register the event with the test framework using **_AFT.addEventToMonitor**
+    function.
 
-> **NOTE**: about now this timeout arguments simply wait a number of seconds
-> instead of interrupt as soon as the event has been received.
+    Check if an event has been correctly received. An event name use the
+    application framework naming scheme: **api/event_name**.
+
+* **_AFT.assertLogNotReceived(testName, logMsg)**
```
Comment (claneys) [inline README.md]: To be removed. Log are not managed anymore. :)

File: conf.d/cmake/config.cmake [hunk 8]
```diff
@@ -193,7 +193,7 @@ set(AFB_REMPORT "1234" CACHE PATH "Default binder listening port")
 
 # Print a helper message when every thing is finished
 # ----------------------------------------------------
-set(CLOSING_MESSAGE "Typical binding launch: afb-daemon --name afbd-${PROJECT_NAME} --port=${AFB_REMPORT} --workdir=package --ldpaths=/opt/AGL/lib64/afb:lib --token=\"${AFB_TOKEN}\" --tracereq=common --verbose")
+set(CLOSING_MESSAGE "Typical binding launch: afb-daemon --name afbd-${PROJECT_NAME} --port=${AFB_REMPORT} --workdir=package --ldpaths=/opt/AGL/lib64/afb:lib --token=\"${AFB_TOKEN}\"")
```
Comment (claneys) [inline conf.d/cmake/config.cmake:196]: Why removed those flags ?

File: conf.d/project/etc/test-config.json [hunk 9]
```diff
@@ -15,8 +15,8 @@
         "info": "Launch all the tests",
         "action": "lua://AFT#_launch_test",
         "args": {
-            "trace": "hello",
-            "files": ["helloworld.lua"]
+            "trace": "AFT",
```
Comment (claneys) [inline conf.d/project/etc/test-config.json]: Why change ? If you don't trace "hello" api you will not be able to see its events coming.

**Comment by claneys** [comment]
Ok, je pense que tu peux tout mettre dans un seul commit en rebasant. Et bien faire attention à signer les commit ainsi que d'avoir l'author avec le bon mail.

**Comment by claneys** [comment]
Squashed and Merged at the head of master
